maltd-frontend/src/components/base/ListElement/ListElement.stories.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/jsx-filename-extension */
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import ListElement from "./ListElement";
+
+const listElement = {
+  title: "MALTD",
```

Comment:
maybe a TODO to remove this at a later date? 😝

Reply:
why? it is an open source project right so this placeholder value as a sample project title should be ok right? 😄

Reply:
You're not wrong.